client/components/RelatedProducts/Related/RelatedProductsList.jsx

```diff
@@ -3,14 +3,20 @@ import RelatedProductCard from './RelatedProductCard.jsx';
 import products from '../../DummyData/ProductDummyData';
 import GridList from '@material-ui/core/GridList';
 // import Carousel from 'react-material-ui-carousel'
+import Carousel from 'react-elastic-carousel';
 
+//CSS styling for later so disabled arrow does not show:
+// .rec.rec-arrow:disabled {
+//   visibility: hidden;
+// }
 
 const RelatedProductsList = (props) => {
```

Comment:
Try destructuring your props like so ```const RelatedProductsList = ({ relatedWithNameCatPrice }) => {```

Reply:
@ChrisRPeterson 
done! Thanks for suggesting!